src/api/apiUtils.js

```diff
@@ -3,12 +3,24 @@ import { session } from '../models/auth';
 export async function handleResponse(response) {
   if (response.status === 204) return {};
   if (response.ok) return response.json();
+
+  // server-side validation error occurred.
+  // Server side validation returns a string error message, so parse as text instead of json.
+  const error = await response.text();
+  console.log('handleResponse error ---', error);
+
   if (response.status === 400) {
-    // server-side validation error occurred.
-    // Server side validation returns a string error message, so parse as text instead of json.
-    const error = await response.text();
     throw new Error(error);
   }
+  if (response.status === 500 && error.includes('author')) {
+    // server-side error occurred. Author account not found.
```

Comment:
is checking for the string 'author' enough to guarantee there is only one candidate error that could cause this 500 ?

Reply:
Probably not specific enough, but I reasoned that it would either be that their admin handle didn't exist or some other error that we haven't encountered already when requesting authors that they should talk to us about.

I'll adjust it to the specific SQL error for now and we can worry about other error-handling later

Reply:
Ah! I see the message has been updated.  Before I was just getting the SQL error